CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# How to Contribute
+
+If you're interested in contributing, take a look at the general [contributer's guide](https://github.com/Microsoft/ApplicationInsights-Home/blob/master/CONTRIBUTING.md) first.
+
+## Run Test
```

Comment:
can you add build section first. Is `gem build application_insights.gemspec` enough?